### DISCUSSION TITLE
Problem with Turbulent channel flow case in Incompact3D

### DISCUSSION THREAD

**Discussion by jushi-123** [xcompact3d/Incompact3d#306] [Q&A] [answered]
Hi Everyone,

Recently, I have started using the Incompact3D as a part of the validation exercise, I wanted to run the turbulent channel flow test case and reproduced the result of Moser et al(Retao=180,1999). While running my case, I just used the following BC-Channel-flow.prm file. After get the average velocity profile and postprocessed it with tecplot , i get the Retao=203.
[BC-Channel-flow.txt](https://github.com/user-attachments/files/17476481/BC-Channel-flow.txt)

Could the admin please look into it?

Regards.

**Comment by slaizet** [comment, marked as answer]
Hi, 
I am not quite sure which version of the code you are using but we stopped using prm files a long time ago. The channel flow test case at Re_tau=180 has been extensively validated with Xcompact3d and we are proving a ready-to-run input file for it (https://github.com/xcompact3d/Incompact3d/blob/master/examples/Channel/input_DNS_Re180_LR_explicittime.i3d) for which you only need to change the spatial resolutions to match the data of Moser et al.
Best,
Sylvain

**Reply by jushi-123** [reply]
Hello Dr. Laizet,
I have some doubt in the meaning of variables u1 and u2 in input.i3d. According to the test, u1(u2) is the max(min) velocity for inflow condition. However, the inflow condition for turbulent channel flow should be periodic and i have not found u1 and u2 in the whole program.So what's the function of u1 and u2 as a parameter in input.i3d?

**Reply by slaizet** [reply]
For the periodic channel flow, there is no inflow/outlfow boundary conditions so u1/u2 are not required (maybe we should remove them from the input file to avoid confusion)
Best,
Sylvain
